File: C #/C# Part 1/03.Operators-and-Expressions/10.PointCircleRectangle/PointCircleRectangle.cs
using System;

namespace _10.PointCircleRectangle
{
    class PointCircleRectangle
    {
        static void Main()
        {
            double x = double.Parse(Console.ReadLine());
            double y = double.Parse(Console.ReadLine());
            double r = 1.5;
            bool circle = (x - 1) * (x - 1) + (y - 1) * (y - 1) <= (r * r);
            bool rectangle = (x >= -1 && x <= 5) && (y >= -1 && y <= 1);

            if ((circle == true) && (rectangle == true))
            {
                Console.WriteLine("inside circle" + " " + "inside rectangle");
            }
            else if (!(circle == true) && (rectangle == true))
            {
                Console.WriteLine("outside circle" + " " + "inside rectangle");
            }
            else if ((circle == true) && !(rectangle == true))
            {
                Console.WriteLine("inside circle" + " " + "outside rectangle");
            }
            else if (!(circle == true) && !(rectangle == true))
            {
                Console.WriteLine("outside circle" + " " + "outside rectangle");
            }
        }
    }
}

File: JavaScript/JS Basics/04.Operators and Expressions/09.Point in Circle and outside Rectangle/PointCircleRectangle.js
function solve(args){

  var pointX = Number(args[0]);
  var pointY = Number(args[1]);

  var inCricle = false;
  var inRectangle = false;

  var distanceToCircleX = pointX - 1;
  var distanceToCircleY = pointY - 1;

  var distanceToCircle = Math.sqrt(
    (distanceToCircleX
      * distanceToCircleX)
    + (distanceToCircleY
      * distanceToCircleY));

  if (distanceToCircle <= 1.5) {
    inCricle = true;
  }

  if (pointX >= -1
    && pointX <= 5
    && pointY <= 1
    && pointY >= -1) {

    inRectangle = true;
  }

  var output = '';
  if (inCricle) {
    output += 'inside circle ';
  }
  else {
    output += 'outside circle ';
  }

  if (inRectangle) {
    output += 'inside rectangle';
  }
  else {
    output += 'outside rectangle';
  }
  console.log(output);
}
